### ISSUE TITLE
Add High DPI Awareness

### ISSUE THREAD

**Issue by mortalis13** [doublecmd/doublecmd#284]
Please add High DPI support for Windows and for multiple monitors with different DPI scaling.
According to this page: https://wiki.freepascal.org/High_DPI#High_DPI_in_Lazarus_1.8_and_above.

Current settings:
- Monitor 1, scale 100%, 96 dpi
- Monitor 2, scale 150%, 144 dpi

The program opens in the Monitor 1 by default.
When the window is moved to the Monitor 2, the texts, icons etc. are blurred.

**Comment by alexx2000** [comment]
DC already has High DPI support, but without per monitor DPI awareness.

**Comment by mortalis13** [comment]
OK, then the current monitor DPI should be used when moved between the monitors, please.
I posted that link, because the current project settings make the window blurry when moved from the main monitor to the secondary.
But when changing the Application options to the recommended, the blurry issue is solved.
![image](https://user-images.githubusercontent.com/10464927/144817024-f594e110-706f-4f4b-a808-89dccf7d89e1.png)

But the scaling (pixmaps, fonts) is incorrect.
So basically I would like the scaling to be fixed.

I found the `Screen.PixelsPerInch` used in `ucolumns.pas` for example, when it calculates `AColumn.Width`. That always gives **96**.
But not sure how to get the active monitor DPI (which should change bewteen 96 and 144 in my case) and what other places should be changed to make the scaling correct.

**Comment by mortalis13** [comment]
Another question.

I'm trying to get the active monitor by a TPoint.
But when the app window is on the main monitor the Left coordinate is **-8**.

I've put a debug message in the `TfrmMain.WMMove`
`dcdebug('..Left: ' + IntToStr(Left));`

Why is that?

**Comment by alexx2000** [comment]
The main problem is icons. Icons loaded with scaling for primary monitor and cached. Icons size cannot be changed on the fly. That's why I did not enable per monitor support.
> the Left coordinate is -8.

It is normal when window maximized.
